client/signup/steps/jpo-contact-form/style.scss

```diff
@@ -1,14 +1,49 @@
+.jpo__contact-form-wrapper {
+	margin-left: auto;
+	margin-right: auto;
+	display: flex;
+	max-width: 325px;
+}
+
 .jpo__contact-form-wrapper .card {
 	max-width: 320px;
 	text-align: center;
+}
+
+.jpo-contact-form__description {
+	font-size: 1.25rem;
+	color: $gray;
+}
+
+.jpo-contact-form__image {
+	padding:15px;
+}
+
+.jpo-contact-form__choice-copy {
+	border-top: 1px solid rgba(200, 215, 225, 0.5);
+	padding: 15px;
+	border-top: 1px solid transparentize( lighten( $gray, 20% ), .5 );
```

Comment:
This is overriding the property in line 23. Usually, if what you see is ok, you have to delete the first one, in this case, the one in line 23.